test/build_size_test.sh

```diff
@@ -12,7 +12,7 @@ set -e
 source "$(dirname "${BASH_SOURCE[0]}")/../.ci/scripts/utils.sh"
 
 # TODO(#8357): Remove -Wno-int-in-bool-context
-COMMON_CXXFLAGS="-fno-exceptions -fno-rtti -Wall -Werror -Wno-int-in-bool-context"
+COMMON_CXXFLAGS="-fno-exceptions -fno-rtti -Wall -Werror -Wno-int-in-bool-context -Wno-stringop-overread"
```

Comment:
Ah this, I thought I removed the memcpy which trigged this for Arm baremetal build.